modules/learn/pages/clusters-and-availability/rebalance-and-index-service.adoc

```diff
@@ -0,0 +1,815 @@
+
+Shard affinity keeps all index partitions with the same shard ID on the same node and moves them as a single unit during index rebalancing.
+This behavior also applies to internal data structures associated with those shards, which remain collocated and move together.
+
+For information about how this affects File-Based Rebalance, see <<#shard-affinity-and-fbr,Relationship Between Shard Affinity and File-Based Rebalance>>.
+
+[#index-rebalance-methods]
+== Index Rebalance Methods
+
+Couchbase Server provides the following index rebalance methods:
+
+* *Standard Rebalance*: This is the default DCP-based method used for self-managed clusters.
+This method reads data from the Data Service and rebuilds indexes on the destination node.
+Couchbase Server uses this method when File-Based Rebalance is disabled.
++
+For an example of DCP-based rebalance operation, see <<#full-swap-rebalance,Full Swap Rebalance (DCP-based Rebuild)>>.
+
+* *<<#file-based-rebalance-fbr,File-Based Rebalance or Shard Based Rebalance>>*: This is the relocation method.
+In this method, Couchbase Server moves index files between nodes instead of rebuilding them during a rebalance.
+Copying the index files is faster than having the target node rebuild the index from scratch.
++
+NOTE: For Couchbase Capella (Provisioned Clusters), File-Based Rebalance is enabled by default.
+It can be disabled through a support request.
+The on and off behavior is the same as in self-managed clusters.
+
+[#file-based-rebalance-fbr]
+=== File-Based Rebalance or Shard Based Rebalance
+[.edition]#{enterprise}#
+
+File-Based Rebalance, called also as Shard Based Rebalance, moves index files between nodes instead of rebuilding them.
+
+File-Based Rebalance is composed of the following features:
+
+* xref:learn:clusters-and-availability/rebalance-and-index-service.adoc#shard-affinity[*Shard Affinity*]: It decides shard placement, so partitions and their replicas share the same slot ID.
+* *Shard Based Rebalance:* It moves the shard files between nodes during rebalance.
+
+[NOTE]
+====
+* File-Based Rebalance is supported only if you have enabled Standard Global Secondary xref:manage:manage-settings/general-settings.adoc#index-storage-mode[Index Storage Mode] on your cluster.
+
+* You cannot use File-Based Rebalance if you have enabled xref:learn:services-and-indexes/indexes/storage-modes.adoc#memory-optimized-index-storage[Memory Optimized Index Storage] in the xref:manage:manage-settings/general-settings.adoc#index-storage-mode[Index Storage Mode] settings on your cluster because it does not store index metadata in files.
+* Shard Based Rebalance and Rebalance Based on File Transfer are synonyms for File-Based Rebalance.
+====
+
+For examples of Index Rebalance and File-Based Rebalance in action, see <<#fbr-all-examples,Understanding Index Rebalance with Examples>>.
+
+[#shard-affinity-and-fbr]
+==== Relationship Between Shard Affinity and File-Based Rebalance
```

Comment:
As mentioned earlier, @Nischal1729, let's double check this. 

In general, the relation between shard based rebalance and file transfer is not just about partitions. It is also important to ensure the HA during index movements.